fail to build hexo

### Check List

- [X] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting).
- [X] I have already searched existing issues and they are not help to me.
- [X] I examined error or warning messages and it's difficult to solve.
- [X] I am using the [latest](https://github.com/hexojs/hexo/releases) version of Hexo. (run `hexo version` to check)
- [X] My Node.js is matched [the required version](https://hexo.io/docs/#Required-Node-js-version).

### Expected behavior

Build successfully

### Actual behavior

> https://github.com/hexojs/hexo/actions/runs/7666372926/job/20894038821?pr=5407

![fail](https://github.com/hexojs/hexo/assets/22849383/356a5117-5b40-4dc2-94ab-df9bf163c71a)

```
> hexo@7.1.1 build D:\hexo
> tsc -b

node_modules/.pnpm/chokidar@3.5.3/node_modules/chokidar/types/index.d.ts:9:14 - error TS2420: Class 'import("D:/hexo/node_modules/.pnpm/chokidar@3.5.3/node_modules/chokidar/types/index").FSWatcher' incorrectly implements interface 'import("fs").FSWatcher'.
  Type 'FSWatcher' is missing the following properties from type 'FSWatcher': ref, unref

9 export class FSWatcher extends EventEmitter implements fs.FSWatcher {
               ~~~~~~~~~


Found 1 error.

 ELIFECYCLE  Command failed with exit code 1.
```

### How to reproduce?

1. git clone
2. npm i
3. npm run build

### Is the problem still there under `Safe mode`?

..

### Your Node.js & npm version

_No response_

### Your Hexo and Plugin version

_No response_

### Your `package.json`

_No response_

### Your site's `_config.yml` (Optional)

_No response_

### Others

--- a/package.json
+++ b/package.json
@@ -69,7 +69,7 @@
     "@types/abbrev": "^1.1.3",
     "@types/bluebird": "^3.5.37",
     "@types/js-yaml": "^4.0.9",
-    "@types/node": "^18.11.8",
+    "@types/node": "^18.11.8 <18.19.9",
     "@types/nunjucks": "^3.2.2",
     "@types/text-table": "^0.2.4",
     "0x": "^5.1.2",

